tools/download-wheels.py

```diff
@@ -1,9 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 """
-Download SciPy wheels from Anaconda staging area.
-
+Download wheels from Anaconda staging area.
```

Comment:
A quick note on generalizing here--if I remember correctly this module was copied very closely from the original upstream NumPy version drafted by @charris.

We probably don't need to worry too much about that, but just figured I'd ping Chuck in case there's a preference on making the most general version the one that is farthest upstream.

I've also noticed the module struggles behind a proxy (I manually edit a tiny bit when working behind a proxy), but that's another matter.

Reply:
I started with `terryfy/wheel_uploader` and simplified it for NumPy and Anaconda. I think it is easy to adapt for other Anaconda repos and for NumPy prefer its current simplicity over generality, using argument defaults to replace constants still leaves it somewhat complicated for general users. Maybe leave the constants in and use them as the parser defaults, that would make changing the defaults more obvious. @matthew-brett How do you intend this to be used?

Reply:
Note that there are some tweaks in the NumPy version to make the output prettier and more informative.

Reply:
@tylerjereddy What sort of changes are needed behind a proxy? Some documentation might be useful.